playwright.config.ts

```diff
@@ -0,0 +1,105 @@
+import { defineConfig, devices } from '@playwright/test';
+
+/**
+ * Read environment variables from file.
+ * https://github.com/motdotla/dotenv
+ */
+// require('dotenv').config();
+
+/**
+ * See https://playwright.dev/docs/test-configuration.
+ */
+export default defineConfig({
```

Comment:
For a more generic setup it is recommended to add a global-setup.ts file, that store variables such as the URL.

Reply:
This was autogenerated by `npm init playwright@latest` as recommended of [their homepage](https://playwright.dev/docs/intro). Would you say this can be done in a follow-up PR?